lib/hand.rb

```diff
@@ -10,77 +20,59 @@ def add_card(card)
     @cards.push(card)
   end
 
-  def contains_card?(card)
-    @cards.include?(card)
+  def length
+    @cards.length
   end
 
-  def high_card
-    @cards.max_by { |card| card.rank(Deck.FACE_RANKS) }
-  end
-
-  def contains_pair?
-    card_occurences.value?(2)
-  end
-
-  def contains_three_of_kind?
-    card_occurences.value?(3)
-  end
-
-  def contains_four_of_kind?
-    card_occurences.value?(4)
-  end
-
-  def contains_full_house?
-    contains_pair? && contains_three_of_kind?
-  end
-
-  def contains_two_pair?
-    card_occurences.select { |_k, v| v == 2}.count == 2
-  end
-
-  def contains_straight?
-    card_sum == consecutive_sum || card_sum == 28 && all_unique?
-  end
-
-  def contains_flush?
-    suit_occurences.value?(5)
+  def contains_card?(testcard)
```

Comment:
I think the name `testcard` is a bit confusing. How about just using `card` here? You'd have to use something else for the card in the hand with which you're comparing it to on line 28, though. Maybe call that one something like `existing_card` or `card_in_hand`?